.github/workflows/plugin-packages.yml

```diff
@@ -143,6 +145,18 @@ jobs:
       - name: Test Log-Collector Plugin Locally
         run: make test-log-collector-plugin-locally
 
+      - name: Set up Cloud SDK
+        if: ${{ github.ref == 'refs/heads/main' }}
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          project_id: ${{ secrets.GCS_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCS_ARTIFACTS_CREDENTIALS }}
+          export_default_credentials: true
+
+      - name: Upload Preflight Plugin Packges to github artifact
```

Comment:
we are not uploading to github artifact right? change the `name`

Reply:
done